distribution/changelog.txt

```diff
@@ -24,6 +24,7 @@
 - Fix: [#12881] Guests' favourite rides are not listed in the guest window.
 - Fix: [#12910] Plugin API: getRide sometimes returns null for valid ride IDs.
 - Fix: [#12912] Plugin: selectedCell of CustomListView is being ignored on creation.
+- Fix: [#12918] Cannot place "Blue Hurricane" (hypercoaster).
```

Comment:
I realise this is a the title of the original issue, but I think it would be better if all of the "Hypercoasters, Monster Trucks, Classic Mini Roller Coasters, Spinning Wild Mouses and Hyper-Twisters" track types would be listed. As I understand it:
```suggestion
- Fix: [#12918] Cannot place vanilla TD6 tracks of the Hypercoasters, Monster Trucks, Classic Mini Roller Coasters, Spinning Wild Mouses and Hyper-Twisters types.
```